src/mmrelay/matrix/credentials.py

```
@@ -0,0 +1,310 @@
+                    device_id=e2ee_device_id,
+                    credentials=credentials,
+                    credentials_path=credentials_path,
+                )
+            else:
+                facade.logger.error(
+                    "Automatic login failed. Please check your credentials or use 'mmrelay auth login'"
+                )
+                return None
+        except asyncio.CancelledError:
+            raise
+        except (OSError, IOError) as e:
```

Ruff (0.15.9)

[warning] Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)